lib/metadata/labels.js

```diff
@@ -108,3 +111,28 @@ exports.fromFiles = function(filenames) {
 exports.fromWorkingGroups = function labels(workingGroups) {
     return workingGroups.map(function(shortname) { return "wg-" + shortname; })
 };
+
+/**
+ * Combine multiple arrays of labels into a single array, honoring limitations
+ * of the target platform.
+ *
+ * @param {...Array<string>} labels - any number of arrays of labels
+ *
+ * @returns {Array<string>}
+ */
+exports.merge = function(labels) {
+    var all = [];
+    var length = arguments.length;
+    var index;
+
+    for (index = 0; index < length; index += 1) {
+        all.push.apply(all, arguments[index]);
+    }
+
+    if (all.length > MAX_LABELS) {
+        all.length = 0;
```

Comment:
Isn't this just `all = ["infra", "very-large"]`?

(I'm not sure what the use case is for`"very-large"`)

Reply:
> Isn't this just `all = ["infra", "very-large"]`?

It is

> (I'm not sure what the use case is for `"very-large"`)

That label demonstrates that this truncation took place and differentiates these pull requests from others that change only files in the `tools/` directory.

Reply:
Since most PRs that touch too many files are rebases gone wrong, being able to search for them might provide some small value, so a label for it sounds good. I currently sometimes look for those mishaps by looking for things labeled with multiple common labels :)